package.json

```diff
@@ -154,13 +157,13 @@
             }
         ]
     },
-    "dependencies": {},
     "devDependencies": {
         "@typescript-eslint/eslint-plugin": "5.7.0",
```

Comment:
Do you also want to bump the extension version, or do we have more changes to do before the next release?

Reply:
Updated.